code/src/components/AddItemForm.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from 'reducers/todos'
+import '../index.css'
+
+export const AddItemForm = () => {
+    const [name, setName] = useState('')
+    const [category, setCategory] = useState('')
+    
+    const dispatch = useDispatch()
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
```

Comment:
There is not logic implemented to check if value of name or category is empty string - should be fixed

Reply:
This is still not fixed

Reply:
i don´t know how to fix this. So long ago we did this so I don't remember what to do here.

Reply:
You simply have to check if `event.target.value` is an empty string

Reply:
i don´t know what that means, only that I have not learn this all the way.

Reply:
You did it many times already Nathalie :)
All logic inside this funtion should be wrapped in some sort of if statement - that should help you understand what you need to do